README.rst

```diff
@@ -104,3 +130,14 @@ Apply the changes:
 You should have requested number of resources spawned on Openstack, and ansible_inventory file produced as output for Kayobe.
 
 Copy your generated id_rsa and id_rsa.pub to ~/.ssh/ on Ansible control host if you want Kayobe to automatically pick them up during bootstrap.
+
+Configure Ansible Control Host
+
+Using the `deploy-oc-networks.yml` playbook you can setup the ansible control host to include the kayobe/kayobe-config repositories with `hosts` and `admin-oc-networks`.
+It shall also setup the kayobe virtual environment, allowing for imediate configure and deployment of OpenStack.
+
+You must first run
+
+.. code-block:: console
+   ansible-galaxy install -r ansible/requirements.yml
```

Comment:
There's a bit of a bootstrapping issue here - we need to install ansible. In my mind the host that you run terraform from would typically be the ansible control host. Is there a reason not to do that?

Reply:
One good reason is that if you wanted to run `terraform destroy; terraform apply`, it would wipe out any changes you made to the config.

Reply:
The tear-down.sh will help prevent wiping out any config changes. Also the deploy-openstack-config.yml playbook will not perform a git pull keeping any local changes safe.